examples/demo/src/index.tsx

```diff
@@ -1,8 +1,11 @@
-import 'proxy-polyfill';
```

Comment:
this one should still be required?

Reply:
https://caniuse.com/proxy

Reply:
Now I understand that upgrading to React 18 means dropping support for IE

https://react.dev/blog/2022/03/08/react-18-upgrade-guide#dropping-support-for-internet-explorer

We should make it clear in the upgrade guide. Also, there are many ie11 polyfills in our codebase, this PR should remove them.

Reply:
Will do in a dedicated PR